examples/data-sources/prefect_account_role/data-source.tf

```diff
@@ -1,9 +1,14 @@
-# Read down the default Owner Account Role
-data "prefect_account_role" "owner" {
+# Read down the default Admin Account Role
+data "prefect_account_role" "admin" {
   name = "Admin"
 }
 
 # Read down the default Member Account Role
 data "prefect_account_role" "member" {
   name = "Member"
 }
+
+# Read down the default Owner Account Role
+data "prefect_account_role" "owner" {
+  name = "Owner"
+}
```

Comment:
is pre-commit running by any chance

Reply:
yes, but im getting an odd failure on this `internal/provider/customtypes/timestamp_type.go:18:6:`

Reply:
Yeah that's probably https://github.com/PrefectHQ/terraform-provider-prefect/issues/231, it's just a deprecation and not a removal but for some reason `pre-commit` treats it as a failure. I'll work on a PR to update that.